gossip/src/crds_value.rs

```diff
@@ -600,6 +597,9 @@ impl RestartLastVotedForkSlots {
     // This number is MAX_CRDS_OBJECT_SIZE - empty serialized RestartLastVotedForkSlots.
     const MAX_BYTES: usize = 824;
 
+    // Per design doc, we should start wen_restart within 7 hours.
+    pub const MAX_SLOTS: usize = 65535;
```

Comment:
does this need to be `pub`?
also `u16::MAX as usize` or `0xFFFF` might be more descriptive where this number comes from.

Reply:
Changed. It needs to be pub because later when we create the fork from wen_restart code we can limit the fork size at 65535.